src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodsApiServiceSpec.scala

```diff
@@ -42,18 +42,36 @@ class MethodsApiServiceSpec extends ServiceSpec with MethodsApiService {
   }
 
   "MethodsService" - {
-    "when testing all HTTP methods on the methods path" - {
-      "MethodNotAllowed error is not returned" in {
-        httpMethods map {
```

Comment:
this unit test was just plain wrong, and only passed due to funky behavior in `passthroughAllPaths`. The test now tests reality.